### ISSUE TITLE
Omer & Cyrus please do something

### ISSUE THREAD

**Issue by dizmac** [Spxxk/Whodunit#3]
Hi Omer,

We need to talk about the level of your contribution to this collaborative project. As you can see in the image below, you have deleted more code than you have created. Moreover, apart from adding the game files, you have made barely any meaningful commits, as shown in the second image, and the commit history. I could only find maybe 1-2 bug-ridden commits that were somewhat useful. This is a serious issue, please do more work, or you will be removed from the group.
![image](https://user-images.githubusercontent.com/57689155/234909895-a8233c2e-8690-483e-a9a4-fe6725b4efae.png)
![image](https://user-images.githubusercontent.com/57689155/234910540-5dea9575-5f28-4db3-9820-ef5e005474df.png)

**Comment by Spxxk** [comment]
I fully agree with my colleague, Omer Hussein please commit some code that will impact our game you are on par with Cyrus right now.

**Comment by dizmac** [comment]
@CyrusA41 please also do something

**Comment by dizmac** [comment]
![image](https://user-images.githubusercontent.com/57689155/234950774-4ae59b52-ffb4-4976-8a53-0e201fa8f719.png)
Uh Cyrus, where are you on the list?
